frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -537,10 +535,6 @@ class RegistrationForm extends Component {
     this.setState({ isCheckboxChecked: !this.state.isCheckboxChecked });
   };
 
-  componentDidMount = () => {
```

Comment:
Should this one be kept? or is its removal part of the fix? Mostly just curious

Reply:
It's part of the fix. Before the default value was _false_, but needed to be _true_ to NOT display the error in the page title. That why I was using a _componentDidMount()_ function, to put the value to _true_ as soon as we load the page.

The whole logic of that was not really good. That why I wanted to fix that.